modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -1,18 +1,28 @@
 = Perform Runtime Fabric Cluster Upgrades
 
-Anypoint Runtime Fabric is powered by a set of components including Docker, Kubernetes, and the Ops Center. These components run on virtual machines or bare-metal servers that act as nodes on the cluster. To perform an upgrade to these cluster-level components and apply updates to node configurations, follow the procedure below.
+Anypoint Runtime Fabric is powered by a set of components including Docker, Kubernetes, and the Ops Center. These components run on virtual machines or bare-metal servers that act as nodes on the cluster. Perform the following steps to upgrade these cluster-level components and apply updates to node configurations.
 
 == Before You Begin
 
+* Run the upgrade procedure on a non-production Runtime Fabric to verify compatibility with your environment and to gain familiarity with the procedure.
+* Plan the upgrade procedure when your team does not need to deploy new applications or managing existing applications on Runtime Fabric.
 
-* Run the upgrade procedure on a non-production Runtime Fabric first to verify compatibility with your environment and to gain familiarity with the procedure.
-* Run the upgrade procedure when your team does not need to deploy new applications or managing existing applications on Runtime Fabric.
-+
-The upgrade can take over an hour to complete.
+** The upgrade can take over an hour to complete.
+** The connection between Runtime Fabric and Anypoint Runtime Manager might become disconnected multiple times during the upgrade procedure. Connectivity will be restored after the upgrade has completed.
```

Comment:
new style guidance says "no" to modal verbs: https://confluence.internal.salesforce.com/pages/releaseview.action?spaceKey=DOCTEAM&title=modal+verbs

Reply:
Fixed - thank you!